codegen/templates/endpoint_register.tmpl

```diff
@@ -12,6 +12,9 @@ import (
 	{{end}}
 
 	"github.com/uber/zanzibar/runtime"
+
+	// TODO: (lu) remove this, to be generated
+	"github.com/uber/zanzibar/examples/example-gateway/endpoints/baz"
```

Comment:
This approach doesn't work with edge-gateway, we actually broke clients.go with edge-gateway.